packages/ui/src/index.ts

```diff
@@ -15,6 +15,7 @@ export * from "./Buttons";
 export * from "./DataTable";
 export * from "./Table";
 export * from "./utils";
+export * from "./FilesTable";
```

Comment:
Let maintain the order here. Also put a empty line between export and import blocks.